frontend/src/pages/org/settings/components/profile.ts

```diff
@@ -0,0 +1,159 @@
+import { localized, msg } from "@lit/localize";
+import { serialize } from "@shoelace-style/shoelace/dist/utilities/form.js";
+import { html } from "lit";
+import { customElement } from "lit/decorators.js";
+
+import { BtrixElement } from "@/classes/BtrixElement";
+import { columns, type Cols } from "@/layouts/columns";
+import { RouteNamespace } from "@/routes";
+import { formValidator, maxLengthValidator } from "@/utils/form";
+
+@localized()
+@customElement("btrix-org-settings-profile")
+export class OrgSettingsProfile extends BtrixElement {
+  private readonly validateDescriptionMax = maxLengthValidator(150);
+
+  render() {
+    const orgBaseUrl = `${window.location.protocol}//${window.location.hostname}${window.location.port ? `:${window.location.port}` : ""}`;
+
+    const cols: Cols = [
+      [
+        html`
+          <label for="orgVisibility" class="form-label text-xs">
+            ${msg("Visibility")}
+          </label>
+          <div>
+            <sl-switch
+              id="orgVisibility"
+              name="enablePublicProfile"
+              size="small"
+              ?checked=${this.org?.enablePublicProfile}
+            >
+              ${msg("Allow anyone to see org")}
```

Comment:
I feel like this might be better off as a select element — but barring that, maybe this helps the label feel a little better?
```suggestion
              ${msg("Allow anyone to view org")}
```